src/lib/swapper/swappers/ThorchainSwapper/utils/makeMemoWithShortenedFinalAssetAmount.test.ts

```diff
@@ -0,0 +1,77 @@
+import { describe, expect, it } from 'vitest'
+
+import { MEMO_PART_DELIMITER } from './constants'
+import { makeMemoWithShortenedFinalAssetAmount } from './makeMemoWithShortenedFinalAssetAmount'
+
+const RECEIVE_ADDRESS = '0x32DBc9Cf9E8FbCebE1e0a2ecF05Ed86Ca3096Cb6'
+// 178 bytes
+const BIG_ADDRESS =
+  '0x32DBc9Cf9E8FbCebE1e0a2ecF05Ed86Ca3096Cb632DBc9Cf9E8FbC2ecF05Ed86Ca3096cF05Ed86Ca3096Cb60x32DBc9Cf9E8FbCebE1e0a2a3096C2ecF05Ed86Ca3096Cb632DBc9Cf9E8FbCebE1e0a2ecF05Ed86Ca3096Cb6'
+// 354 bytes
+const REALLY_BIG_ADDRESS =
+  '0x32DBc9Cf9E8FbCebE1e0a2ecF05Ed86Ca3096Cb632DBc9Cf9E8FbC2ecF05Ed86Ca3096cF05Ed86Ca3096Cb60x32DBc9Cf9E8FbCebE1e0a2a3096C2ecF05Ed86Ca3096Cb632DBc9Cf9E8FbCebE1e0a2ecF05Ed86Ca3096Cb632DBc9Cf9E8FbCebE1e0a2ecF05Ed86Ca3096Cb632DBc9Cf9E8FbC2ecF05Ed86Ca3096cF05Ed86Ca3096Cb60x32DBc9Cf9E8FbCebE1e0a2a3096C2ecF05Ed86Ca3096Cb632DBc9Cf9E8FbCebE1e0a2ecF05Ed86Ca3096Cb6'
+const AGGREGATOR_TWO_LAST_CHARS = 'd2'
+
+const SHORTENED_FINAL_ASSET_ADDRESS = 'a6df4741'
+
+const affiliateBps = '100'
+const expectedL1AmountOut = '42'
+
+describe('makeMemoWithShortenedFinalAssetAmount', () => {
+  it('should be 80 bytes if maxMemoSize is 80', () => {
+    const quotedMemo = `=:e:${RECEIVE_ADDRESS}:${expectedL1AmountOut}:ss:${affiliateBps}:${AGGREGATOR_TWO_LAST_CHARS}:${SHORTENED_FINAL_ASSET_ADDRESS}`
+
+    const modifiedMemo = makeMemoWithShortenedFinalAssetAmount({
+      maxMemoSize: 80,
+      memoWithoutMinAmountOut: quotedMemo,
+      finalAssetLimitWithManualSlippage: '950875902132134123',
+    })
+
+    expect(modifiedMemo.length).toBe(80)
+    expect(modifiedMemo).toBe(
+      `=${MEMO_PART_DELIMITER}e${MEMO_PART_DELIMITER}${RECEIVE_ADDRESS}${MEMO_PART_DELIMITER}${expectedL1AmountOut}${MEMO_PART_DELIMITER}ss${MEMO_PART_DELIMITER}${affiliateBps}${MEMO_PART_DELIMITER}${AGGREGATOR_TWO_LAST_CHARS}${MEMO_PART_DELIMITER}${SHORTENED_FINAL_ASSET_ADDRESS}${MEMO_PART_DELIMITER}95087590209`,
+    )
+  })
+
+  it('should be any bytes if maxMemoSize is Infinity', () => {
```

Comment:
Test name looks off - if we're not testing the bytes length, what are we actually testing here?
Perhaps we could ensure to test with a memo that would result in quite a large length (i.e > 80, or even >220 if that's possible) and add a sanity test for said length?

Reply:
Added 3 more tests and remove this one